pkgs/applications/office/beebeep/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, mkDerivation, pkgs, stdenv, fetchzip, wrapQtAppsHook }:
+
+mkDerivation rec {
+  pname = "beebeep";
+  version = "5.8.4";
+
+  src = fetchzip {
+    url = "https://netix.dl.sourceforge.net/project/beebeep/Linux/beebeep-${version}-qt5-amd64.tar.gz";
+    sha256 = "sha256-Bi9vsPYB/jmtSLLnj2L0rOeDYz8epwNJs60kgU9kDYA=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+  ];
+
+  buildInputs = with pkgs; [
```

Comment:
```suggestion
  buildInputs = [
```

Reply:
If I remove both `pkgs` references, I cannot access any dependencies.

Reply:
Did it mean I should list all of the dependencies inside the imports?